calcom/cal.com
# adds team prefix to url when creating team

## What does this PR do?

Fixes url prefix when creating a new team.

Fixes #6202 

**Environment**: Production

<img width="620" alt="image" src="https://user-images.githubusercontent.com/26844/209883183-7a3c268c-97a6-4e45-9200-162d2192cab5.png">

## Type of change


## How should this be tested?

* Click Create Team
* Notice url prefix

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
